docs/dc-apps-performance-toolkit-user-guide-confluence.md

```diff
@@ -415,6 +415,12 @@ The **Master (admin) password** will be used later when restoring the SQL databa
 | Permitted IP range | 0.0.0.0/0 _(for public access) or your own trusted IP range_ |
 | Key Name | _The EC2 Key Pair to allow SSH access. See [Amazon EC2 Key Pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) for more info._ |
 
+**Application tuning**
```

Comment:
Estimated index time need to change to 2-3 hours.

Reply:
Also, pls raise separate PR to instance type change and estimated cost table.

Reply:
Will be done